Dangerfile

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+# Warns when there is a big PR
+warn('Big PR') if git.lines_of_code > 500
+
```

Comment:
All checks below now duplicate what we have in `spec/codebase/codebase_spec.rb`. So we should revisit `codebase_spec.rb` to see what we need to keep and remove.

Reply:
We need to discuss more about this.
The `spec/codebase/codebase_spec.rb` is used to check on the test, we can use it on local. But danger will run on CI, if we remove the duplicated tests out of codebase, we have to run the checks manually.